projects/1.20/assets/mekanism-science/mekanismscience/lang/zh_cn.json

```
@@ -0,0 +1,100 @@
+{
+  "advancements.mekanismscience.californium.description": "精炼锎，它在所有有实际用途的元素中原子序数居于顶点",
+  "advancements.mekanismscience.californium.title": "尖端科学",

```

看日文原文“到达点”似乎还是有点差异？
（不过我不会日语，可能望文生义了）

---

到达点我其实觉得很难翻，因为见得多的都是直接不变字拿过来用，可能有些中二就是了

---

日翻中最大的忌讳就是汉字直译

日英均强调“终点/顶点”
```suggestion
  "advancements.mekanismscience.californium.title": "科学的尖端",
```